tests/voxa-chatbase.spec.js

```diff
@@ -55,20 +57,19 @@ describe('Voxa-Chatbase plugin', () => {
       },
     };
 
-    voxaChatbase(voxaStateMachine, chatbaseConfig);
-    return voxaStateMachine.execute(event)
-      .then((reply) => {
-        expect(spy.called).to.be.true;
-        expect(reply.session.new).to.equal(true);
-        expect(reply.session.attributes.state).to.equal('entry');
-        expect(reply.msg.statements).to.have.lengthOf(1);
-        expect(reply.msg.statements[0]).to.equal('Hello! How are you?');
-      });
+    voxaChatbase(app, chatbaseConfig);
+
+    const reply = await alexaSkill.execute(event);
+
+    expect(spy.called).to.be.true;
+    expect(reply.response.outputSpeech.ssml).to.contains('Hello! How are you?');
```

Comment:
this should probably be `reply.speech` reply is an `AlexaReply` object which implements `IVoxaReply`, using `reply.speech` mean we can reuse this for dialogFlow with no change